Mergin/metadata.txt

```diff
@@ -1,7 +1,7 @@
 ; the next section is mandatory
 [general]
 name=Mergin
-qgisMinimumVersion=3.10
+qgisMinimumVersion=3.16
```

Comment:
Do we need to raise the minimum version?

Reply:
It is currently hard to test new code against now obsolete version on various platforms.

Reply:
Reverted to 3.10 with the belief that it should work also for 3.10. If not - users will report issues and we bump the version.